src/main/java/dinkplugin/notifiers/data/XpNotificationData.java

```diff
@@ -0,0 +1,41 @@
+package dinkplugin.notifiers.data;
+
+import dinkplugin.message.Field;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import net.runelite.api.Experience;
+import net.runelite.api.Skill;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class XpNotificationData extends NotificationData {
+    Map<Skill, Integer> xpData;
+    Collection<Skill> milestoneAchieved;
+    int interval;
+    transient String totalXp;
+
+    @Override
+    public List<Field> getFields() {
+        List<Field> fields = new ArrayList<>(2);
+        fields.add(new Field("Total XP", Field.formatBlock("", totalXp)));
+
+        SortedSet<String> maxed = new TreeSet<>();
+        for (var entry : xpData.entrySet()) {
+            if (entry.getValue() >= Experience.MAX_SKILL_XP) {
+                maxed.add(entry.getKey().getName());
+            }
+        }
+        if (!maxed.isEmpty()) {
+            fields.add(new Field("Maxed Skills", Field.formatBlock("", String.join(", ", maxed))));
```

Comment:
Is `Maxed Skills` too ambiguous here? (i.e., lvl 99 or 200m xp)

Reply:
idk what 200m-ers want honestly

Reply:
I'd say remove the field for now and think about that in a future PR/discussion